R/assortativity.R

```diff
@@ -116,7 +116,8 @@ assortativity.degree <- function(graph, directed = TRUE) { # nocov start
 #'   `TRUE` here to do the natural thing, i.e. use directed version of the
 #'   measure for directed graphs and the undirected version for undirected
 #'   graphs.
-#' @param normalized TBD
+#' @param normalized Boolean, whether to compute the normalized covariance,
+#'  i.e. Pearson correlation. Supply true here to compute the standard assortativity.
```

Comment:
```suggestion
#' @param normalized Boolean, whether to compute the normalized assortativity.
#' The non-normalized nominal assortativity is identical to modularity.
#' The non-normalized value-based assortativity is simply the covariance of the
#' values at the two ends of edges.
```

We should later elaborate on what "normalized" means separately for `assortativity()` and `assortativity_nominal()`. The latter is not a Pearson correlation.